scripts/jest/TestFlags.js

```diff
@@ -88,6 +88,7 @@ function getTestFlags() {
       enableUseSyncExternalStoreShim: !__VARIANT__,
       enableSuspenseList: releaseChannel === 'experimental' || www,
       enableOffscreen: releaseChannel === 'experimental' || www,
+      useModernStrictMode: releaseChannel === 'experimental',
```

Comment:
This is an actual feature flag so you shouldn't need this manual override. Did something fail without it?